test/librarySearch.lean

```diff
@@ -2,6 +2,7 @@ import Mathlib
 
 noncomputable section
 
+set_option maxHeartbeats 400000 in
```

Comment:
Maybe don't import all of Mathlib so that this test doesn't start failing if Mathlib grows?

Reply:
Done, good suggestion.